### DISCUSSION TITLE
Hvplot chart not showing up in HTML export

### DISCUSSION THREAD

**Discussion by peterkobylarz** [earthlab-education/Earth-Analytics-2023-01-Intro#504] [Q&A06 Vector Data]
@eculler @nquarder 
I'm trying to export my vector fire project to HTML after completing the analysis. The hvplot of the fire size/count displays in my notebook but does not appear in the exported HTML. Here is the code I'm using to export

```
%%bash
jupyter nbconvert CA_Wildfires.ipynb --to html --no-input
```

[Here's the link to my work in progress project that does not correctly render the hvplots.](https://peterkobylarz.github.io/projects/ca_fires.html)

I'm also curious how to clean up the warning text that appears at the bottom of the HTML page. I tried various warning ignore commands that we have used in the past but they aren't working.

Any ideas would be appreciated!

**Comment by annieM459** [comment]
I am also having this issue

**Reply by ellenalamont17** [reply]
Yes, I had the same issue two weeks ago #476 and still not yet resolved.

**Comment by jsmi374** [comment]
I too am having this same issue with the Dynamic Map hvplot not showing up. In previous weeks, I was able to find a workaround for this. However, this one appears to be rendering in the wrong format when exported to HTML. 

In the Jupyter Notebook, the plot works fine when rendered with the Jupyter IPyWidget renderer.  If you change the presentation style of the plot ( click three dots next to plot) to Text/plain -- VScode builtin output renderer,  you will get the same string as you see in your portfolio page instead of the plot. If there is a way to force the Jupyter IPyWidget renderer to export to html as opposed to the text/plain vscode .... this may solve our problem. I have had no luck on this front though, and I may be off on this

@nquarder @eculler @peterkobylarz @annieM459 @ellenalamont17

**Comment by eculler** [comment]
Hi folks! I still don't have a great solution for this, unfortunately.

I do, however, have a workaround. There are two parts. First, you will need to save your plot directly as a separate html file:

```python
hv.save(your_plot, 'plot.html')
```

You can then link to the html file in a Markdown cell to get it to show up in your portfolio.

**Reply by annettebalmes** [reply]
I have actually been using this workaround as my main method in my portfolio.

What I have done is based off of this post I found online about [embedding bokeh plots in codespaces](https://p-mckenzie.github.io/2017/12/01/embedding-bokeh-with-github-pages/)

Basically you just use hv.save and save each individual plot you make, upload those files into your repository for your website, and then use iframe like so and set the location it can find your html plot, and then you will likely have to adjust the width and height setting to get it to appear exactly as you want in markdown.

```python
<iframe src="/assets/wildfire/hu4_watersheds_colorado.html"
    sandbox="allow-same-origin allow-scripts"
    width="710"
    height="710"
    scrolling="no"
    seamless="seamless"
    frameborder="0">
</iframe>
```

Here is an image of how I used it in my markdown files in my personal website repository, you just put the iframe in between your text:
![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/110803009/61146f80-5e59-4a1e-992d-f970ca409513)

And here is my portfolio website if you want to see the output of my markdown file: 
https://annettebalmes.github.io/general/2023/10/17/wildfire-analysis/
